specification/hello-world.yaml

```diff
@@ -0,0 +1,134 @@
+# This is an OpenAPI Specification (https://swagger.io/specification/)
+# for the Hello World API
+# owned by NHS Digital (https://digital.nhs.uk/)
+openapi: "3.0.0"
+
+info:
+  version: "0.0.1"
+  title: Hello World API
+  description: |
+    ## Overview
+    Use this API alongside our [tutorials](https://digital.nhs.uk/developer/api-tutorials) to learn how to connect to our APIs.
+
+    ## Endpoints
+    To see details for specific endpoints, select from the list on the left hand side of the page:
+    - `GET /hello/world` - an example of an open-access endpoint
+    - `GET /hello/application` - an example of an application-restricted endpoint
+    - `GET /hello/user` - an example of a user-restricted endpoint
+  contact:
+    name: API Management Support
+    email: api.management@nhs.net
+
+servers:
+  - url: 'https://sandbox.api.service.nhs.uk/hello-world'
```

Comment:
Only sandbox should be needed here -- it would be exactly the same in other envs anyway.